docs/getting-started/10-create-function.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Create a Function
+type: Getting Started
+---
+
+Let's now repeat the microservice flow for a Function. This guide shows how you can create a simple Function (`orders-function`) with the same logic as the one in the microservice. In further guides, you will expose the Function, bind it to the Redis storage, and subscribe it to the `order.deliverysent.v1` event type from Commerce mock.
+
+## Reference
+
+This guide demonstrates how [Serverless](/components/event-mesh/) works in Kyma. It allows you to build, run, and manage serverless applications called Functions. You can bind them to other services, subscribe business events from external solutions to them, and trigger the Function's logic upon receiving a given event type.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Apply the [Function CR](/components/serverless/#custom-resource-function) that specifies the Function's logic:
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/orders-service/deployment/orders-function.yaml
+  ```
+
+2. Check that the Function was created and all its conditions are set to `True`:
+
+    ```bash
+    kubectl get functions orders-function -n orders-service
+    ```
+
+    Expect a response similar to this one:
+
+    ```bash
+    NAME                CONFIGURED   BUILT   RUNNING   VERSION   AGE
+    orders-function     True         True    True      1         18m
+    ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+1. Select the `orders-service` Namespace from the drop-down list in the top navigation panel.
```

Comment:
```suggestion
1. From the drop-down list in the top navigation panel, select the `orders-service` Namespace.
```

See [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478392039).